orchestration/dagster_orchestration/hca_utils/utils.py

```diff
@@ -236,21 +175,18 @@ def submit_soft_delete(self, target_table: str, target_path: str) -> str:
         """
         dataset_id = self.get_dataset_id()
 
-        body = {
-            "deleteType": "soft",
-            "specType": "gcsFile",
-            "tables": [
-                {
-                    "gcsFileSpec": {
-                        "fileType": "csv",
-                        "path": target_path
-                    },
-                    "tableName": target_table
-                }
-            ]
-        }
-
-        response = self._hit_jade("applyDatasetDataDeletion", body=body, params={"id": dataset_id})
+        response = self.data_repo_client.apply_dataset_data_deletion(id=dataset_id,
```

Comment:
consider moving this first argument down to its own line, spaced just one indent level out, and lining up the other arguments to that indent level instead.